Dockerfile

```diff
@@ -51,17 +51,47 @@ COPY .gitconfig /.gitconfig
 
 WORKDIR /CIRRUS-core
 
+# Python310 target
+FROM core_base AS python310
+ENV PYTHON_3_10_VERSION "3.10.14"
+RUN \
+        dnf groupinstall "Development Tools" -y && \
+        dnf install openssl-devel bzip2-devel libffi-devel -y && \
```

Comment:
Sure you don't need something like this? https://github.com/docker-library/python/blob/9602fc7493ee0d09c124a05553aadee142992df8/3.8/slim-bullseye/Dockerfile#L48

Reply:
I will add it to be safe.